integration/test/app/workload/workload_cluster_test.go

```diff
@@ -94,7 +94,7 @@ func TestWorkloadCluster(t *testing.T) {
 			{
 				// Bootstrap chart-operator in the giantswarm namespace.
 				AppCRName:     key.ChartOperatorUniqueName(),
-				CatalogName:   key.DefaultCatalogName(),
+				CatalogName:   key.ControlPlaneTestCatalogName(),
```

Comment:
This was a bit hard to find but I guess its OK since the unique chart-operator is installed from the CP catalog.

```
status:
  appVersion: ""
  release:
    reason: 'validation error: user configmap must be named `chart-operator-unique-user-values`
      for app in default catalog'
    status: resource-not-found
```

Reply:
Good catch. 👍